sites/layouts/documentation.html

```diff
@@ -742,7 +742,7 @@ <h3>{% if lng == "fr" %}Ensemble de changements{% else %}Change set{% endif %}</
 	"mapping": [
 		{ "selector": "details > summary", "value": "/name" },
 		{ "selector": "[data-status]", "value": "/status" },
-		{ "selector": "[data-description]", "value": "/description" },
+		{ "selector": "[data-description]", "value": "/description", "isHTML": true },
```

Comment:
Unsure off hand what this is for or what impact rendering it as HTML will have in practice...

Anyone care to chime in?